app/src/main/res/values-land/dimens.xml

```diff
@@ -133,4 +133,9 @@
   <dimen name="feedback_item_question_view_margin_start">64dp</dimen>
   <dimen name="feedback_item_question_view_margin_top">24dp</dimen>
   <dimen name="feedback_item_question_view_margin_end">64dp</dimen>
+
+  <!--  Splash Screen-->
+  <dimen name="splash_screen_logo_width">360dp</dimen>
+  <dimen name="splash_screen_logo_height">640dp</dimen>
+  <dimen name="splash_screen_logo_margin_top">100dp</dimen>
```

Comment:
For this & others: how did you come up with these margins? Also, will these work correctly for both the bitmap & vector drawable (given that they're both used in landscape/other scenarios depending on SDK version).

Reply:
- I figured out these margins just by hit and trial. I used multiple devices (phones & tablets) to find the best fits.
These height and width are in the same ratio as that of the svg logo.

- Currently, these margins are just for the svg drawable. For the bitmap, I haven't set any margin yet. As a result, it is centred vertically as of now. To get the behaviour as shown in the [mock](https://xd.adobe.com/view/dfe8ccd4-39eb-414d-a0b2-b5674357f966-8abb/screen/ddc5589e-10d1-48e4-b551-5487801233e1/) we will have to provide a bottom margin to the bitmap. Not sure but I think different bottom margins will be required for mobile and tablet layouts.

- One thing I wanted to confirm is that is it necessary for the logo (in portrait mode) to sit above the central horizontal axis as shown in the mocks. Like, how bad does the below vertically centred logo look?
![image](https://user-images.githubusercontent.com/51705072/111825507-8a220e80-890d-11eb-9720-6768141fee18.png)
If we place the logo at the centre (as shown above), we can avoid those top and bottom margins.

- Another point is that I think we should update the path data for svg logo so that it only contains the logo and no extra space. Right now it has a viewportWidth=360dp and viewportHeight=640dp but the logo is only in a small portion of this area. This blank area was earlier required when that green background was a part of this svg but we don't need this now. I won't be able to do this part as I don't have any experience in editing svgs.
Also removing the blank area will allow us to use the same dimensions (and margins) for both the svg and the bitmaps.


Mocks for reference:
- Mobile ([Portrait](https://xd.adobe.com/view/dfe8ccd4-39eb-414d-a0b2-b5674357f966-8abb/screen/ddc5589e-10d1-48e4-b551-5487801233e1), [Landscape](https://xd.adobe.com/view/dfe8ccd4-39eb-414d-a0b2-b5674357f966-8abb/screen/375d3300-f70d-44ed-a48a-269a58ecc854))
- Tablet ([Portrait](https://xd.adobe.com/view/dfe8ccd4-39eb-414d-a0b2-b5674357f966-8abb/screen/d39fdc8a-2bee-4867-a1a6-ce4280671c2b), [Landscape](https://xd.adobe.com/view/dfe8ccd4-39eb-414d-a0b2-b5674357f966-8abb/screen/d69bb715-1aea-4815-8401-6e7fde167ed4))

(Ignore the percentage values, I don't think it's possible to set percentage margins in a drawable. Also, please let me know if there's some way to achieve this.)

Reply:
Thanks for the really detailed response @ArpitShukIa!

@mschanteltc do you have any thoughts on how we should relatively position the logo? Should it be center-center? Also, might it be possible to provide us with a new version of the Oppia logo SVG that doesn't include any padding around it? I think we need to reimport this, and convert that exact SVG to WEBP so that we can make sure the bitmap/vector images are exactly the same so that positioning is consistent for both.

Reply:
For ([mobile](https://xd.adobe.com/view/dfe8ccd4-39eb-414d-a0b2-b5674357f966-8abb/screen/375d3300-f70d-44ed-a48a-269a58ecc854/), [tablet](https://xd.adobe.com/view/dfe8ccd4-39eb-414d-a0b2-b5674357f966-8abb/screen/d69bb715-1aea-4815-8401-6e7fde167ed4/)) landscape views, we would vertically center the logo. For ([mobile](https://xd.adobe.com/view/dfe8ccd4-39eb-414d-a0b2-b5674357f966-8abb/screen/ddc5589e-10d1-48e4-b551-5487801233e1/), [tablet](https://xd.adobe.com/view/dfe8ccd4-39eb-414d-a0b2-b5674357f966-8abb/screen/d39fdc8a-2bee-4867-a1a6-ce4280671c2b/)) portrait views, we should place the icon on top of the vertical center. 

The svg in the Android Images folder should already be fit to bounds, so there shouldn't be any padding around it. Is there a specific way I should [export](https://css-tricks.com/wp-content/uploads/2017/04/SVG_Snippet_AI_SaveAs.png) the file? Should I try to convert it to a WEBP file?

Reply:
@BenHenning I have updated the svg logo and created new webp images of different sizes for the new svg. Also, updated the dimens.xml with new values.
The results are pretty much the same as those in the screenshots in the above comments.
Should I send the new screenshots for different screen sizes or will you, as you said, check out the changes locally by yourself?